File: SDKs/googleapis.com/connectors/v2/typescript/src/sdk/models/shared/executeactionresponse.ts
import { SpeakeasyMetadata, SpeakeasyBase } from "../../../internal/utils";



// ExecuteActionResponse
/** 
 * Response message for ActionService.ExecuteAction
**/
export class ExecuteActionResponse extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "json, name=results" })
  results?: Record<string, any>[];
}

File: SDKs/googleapis.com/connectors/v2/python/src/sdk/models/shared/executeactionresponse.py
import dataclasses
from typing import Any,Optional
from dataclasses_json import dataclass_json
from sdk import utils


@dataclass_json
@dataclasses.dataclass
class ExecuteActionResponse:
    r"""ExecuteActionResponse
    Response message for ActionService.ExecuteAction
    """
    
    results: Optional[list[dict[str, Any]]] = dataclasses.field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('results') }})
    
